CMakeLists.txt

```diff
@@ -134,8 +137,6 @@ target_include_directories(${FASTSCAPELIB_TARGET}
     $<INSTALL_INTERFACE:include>
 )
 
-target_link_libraries(${FASTSCAPELIB_TARGET} INTERFACE xtensor)
```

Comment:
Could you confirm this is not needed for downstream projects to implicitly include xtensor headers as a transitive dependency? E.g.,

```cmake
project(my_project)

find_package(fastscapelib REQUIRED)

target_link_libraries(my_target PRIVATE fastscapelib)   # -> also takes care of xtensor dep
```

Reply:
I'll rollback this change, it was related to making `xtensor` a soft dep. not ready yet!